lib/jekyll/assets/proxies/magick.rb

```diff
@@ -171,5 +171,32 @@ def magick_preset_resize(img, cmd)
     # rubocop:enable Style/ParallelAssignment
     # rubocop:enable Metrics/AbcSize
     # ------------------------------------------------------------------------
+
+  end
+
+  Jekyll::Assets::Env.liquid_proxies.add :magick, :img, *(args + presets) do
```

Comment:
Can you update your branch and use `Jekyll::Assets::Env.liquid_proxies.add :magick, [:img, :asset_path], ...` I've made it so that those two duplicated efforts can be merged into a single `add`.  Note: Before you could have looped in a `Array#each`